rti/mosaic-starter/pom.xml

```diff
@@ -162,22 +162,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-resources-plugin</artifactId>
                 <executions>
-                    <execution>
-                        <id>copy-defaults-xml-for-test</id>
-                        <phase>process-test-sources</phase>
-                        <goals>
-                            <goal>copy-resources</goal>
-                        </goals>
-                        <configuration>
-                            <outputDirectory>${project.basedir}/target/test-classes/etc</outputDirectory>
```

Comment:
I don't see the equivalent for this removed block. Where do the tests now get the `etc` files from?

Reply:
Tests also look into `target/classes`, and then in `target/test-classes`. Therefore we don't need to copy them into both locations.